src/aiortc/mediastreams.py

```diff
@@ -51,9 +52,10 @@ def readyState(self) -> str:
         return "ended" if self.__ended else "live"
 
     @abstractmethod
-    async def recv(self) -> Frame:
+    async def recv(self) -> Union[Frame, Packet]:
         """
-        Receive the next :class:`~av.audio.frame.AudioFrame` or :class:`~av.video.frame.VideoFrame`.
+        Receive the next :class:`~av.audio.frame.AudioFrame`, :class:`~av.video.frame.VideoFrame`
+        or :class:`~av.video.frame.VideoFrame`
```

Comment:
Nit: this should be `av.video.packet.Packet` probably

Reply:
Yes. should be `av.packet.Packet`.